PCLumiProd/DerivePCCCorrections_byFill.py

```diff
@@ -109,6 +109,24 @@
 type1corr["5151"]=0.1159
 type1corr["5154"]=0.1205
 type1corr["5161"]=0.1246
+type1corr["5162"]=0.1185
+type1corr["5163"]=0.1229
+type1corr["5169"]=0.1212
+type1corr["5170"]=0.1175
+type1corr["5173"]=0.1249
+type1corr["5179"]=0.1205
+type1corr["5181"]=0.1241
+type1corr["5183"]=0.1243
+type1corr["5187"]=0.1248
+type1corr["5194"]=0.1314
+type1corr["5197"]=0.1236
+type1corr["5198"]=0.1262
+type1corr["5199"]=0.1283
+type1corr["5205"]=0.0866
```

Comment:
Hi Chris,

Actually I also found this very interesting, and was planning to discuss
with you about it. Another thing is that instead of increasing with respect
to the time, the type 1 residual is relatively stable recently (until this
significant drop), I'm not sure what happened there...

On Wed, Aug 31, 2016 at 7:44 PM, capalmer85 <notifications@github.com>
wrote:

> In PCLumiProd/DerivePCCCorrections_byFill.py
> <https://github.com/CMS-LUMI-POG/PCCTools/pull/35#discussion_r77036181>:
>
> > @@ -109,6 +109,24 @@
> >  type1corr["5151"]=0.1159
> >  type1corr["5154"]=0.1205
> >  type1corr["5161"]=0.1246
> > +type1corr["5162"]=0.1185
> > +type1corr["5163"]=0.1229
> > +type1corr["5169"]=0.1212
> > +type1corr["5170"]=0.1175
> > +type1corr["5173"]=0.1249
> > +type1corr["5179"]=0.1205
> > +type1corr["5181"]=0.1241
> > +type1corr["5183"]=0.1243
> > +type1corr["5187"]=0.1248
> > +type1corr["5194"]=0.1314
> > +type1corr["5197"]=0.1236
> > +type1corr["5198"]=0.1262
> > +type1corr["5199"]=0.1283
> > +type1corr["5205"]=0.0866
>
> This is a big drop... what is that all about?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CMS-LUMI-POG/PCCTools/pull/35/files/1d734ecdf299fd1ce8521c770b93c32b27cb57c1#r77036181>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKbGxBWSsMO9qrteXKTpOMRg-dtSRP9cks5qlb1kgaJpZM4JxqZy>
> .
>